src/scenes/Wallet/components/EncryptStatusDialog/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Dialog, { DialogTitle, DialogContent, DialogActions } from 'material-ui/Dialog';
+import { Button } from 'material-ui';
+import { withStyles } from 'material-ui/styles';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import _ from 'lodash';
+
+import styles from './styles';
+import appActions from '../../../../redux/App/actions';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@connect((state) => ({
+  encryptResult: state.App.get('encryptResult'),
+}), (dispatch) => ({
+  encryptWallet: (passphrase) => dispatch(appActions.encryptWallet(passphrase)),
+  clearEncryptResult: () => dispatch(appActions.clearEncryptResult()),
+}))
+export default class EncryptStatusDialog extends Component {
+  static propTypes = {
+    // classes: PropTypes.object.isRequired,
+    encryptResult: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.object,
+    ]),
+    clearEncryptResult: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    encryptResult: undefined,
+  };
+
+  render() {
+    const { encryptResult } = this.props;
+    const isSuccessful = !_.isUndefined(encryptResult) && (encryptResult.includes('wallet encrypted; Qtum server stopping, restart to run with encrypted wallet.'));
+
+    return (
+      <Dialog
+        open={!_.isUndefined(encryptResult)}
+      >
```

Comment:
rule of thumb, if it can fit on one line without making it too difficult to read, put it on one line